[
  {
    "cell": null,
    "code": "B904",
    "end_location": {
      "column": 60,
      "row": 5050
    },
    "filename": "C:\\Users\\wayne\\GitHub\\Python\\Projects\\Ancestry\\action6_gather.py",
    "fix": null,
    "location": {
      "column": 9,
      "row": 5050
    },
    "message": "Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling",
    "noqa_row": 5050,
    "url": "https://docs.astral.sh/ruff/rules/raise-without-from-inside-except"
  },
  {
    "cell": null,
    "code": "B904",
    "end_location": {
      "column": 38,
      "row": 475
    },
    "filename": "C:\\Users\\wayne\\GitHub\\Python\\Projects\\Ancestry\\action9_process_productive.py",
    "fix": null,
    "location": {
      "column": 9,
      "row": 475
    },
    "message": "Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling",
    "noqa_row": 475,
    "url": "https://docs.astral.sh/ruff/rules/raise-without-from-inside-except"
  },
  {
    "cell": null,
    "code": "B904",
    "end_location": {
      "column": 38,
      "row": 615
    },
    "filename": "C:\\Users\\wayne\\GitHub\\Python\\Projects\\Ancestry\\action9_process_productive.py",
    "fix": null,
    "location": {
      "column": 9,
      "row": 615
    },
    "message": "Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling",
    "noqa_row": 615,
    "url": "https://docs.astral.sh/ruff/rules/raise-without-from-inside-except"
  },
  {
    "cell": null,
    "code": "B904",
    "end_location": {
      "column": 85,
      "row": 381
    },
    "filename": "C:\\Users\\wayne\\GitHub\\Python\\Projects\\Ancestry\\core\\browser_manager.py",
    "fix": null,
    "location": {
      "column": 9,
      "row": 381
    },
    "message": "Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling",
    "noqa_row": 381,
    "url": "https://docs.astral.sh/ruff/rules/raise-without-from-inside-except"
  },
  {
    "cell": null,
    "code": "B904",
    "end_location": {
      "column": 14,
      "row": 706
    },
    "filename": "C:\\Users\\wayne\\GitHub\\Python\\Projects\\Ancestry\\database.py",
    "fix": null,
    "location": {
      "column": 13,
      "row": 698
    },
    "message": "Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling",
    "noqa_row": 698,
    "url": "https://docs.astral.sh/ruff/rules/raise-without-from-inside-except"
  },
  {
    "cell": null,
    "code": "B904",
    "end_location": {
      "column": 14,
      "row": 765
    },
    "filename": "C:\\Users\\wayne\\GitHub\\Python\\Projects\\Ancestry\\database.py",
    "fix": null,
    "location": {
      "column": 13,
      "row": 757
    },
    "message": "Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling",
    "noqa_row": 757,
    "url": "https://docs.astral.sh/ruff/rules/raise-without-from-inside-except"
  },
  {
    "cell": null,
    "code": "B904",
    "end_location": {
      "column": 59,
      "row": 992
    },
    "filename": "C:\\Users\\wayne\\GitHub\\Python\\Projects\\Ancestry\\database.py",
    "fix": null,
    "location": {
      "column": 13,
      "row": 992
    },
    "message": "Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling",
    "noqa_row": 992,
    "url": "https://docs.astral.sh/ruff/rules/raise-without-from-inside-except"
  },
  {
    "cell": null,
    "code": "B904",
    "end_location": {
      "column": 10,
      "row": 2243
    },
    "filename": "C:\\Users\\wayne\\GitHub\\Python\\Projects\\Ancestry\\database.py",
    "fix": null,
    "location": {
      "column": 9,
      "row": 2239
    },
    "message": "Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling",
    "noqa_row": 2239,
    "url": "https://docs.astral.sh/ruff/rules/raise-without-from-inside-except"
  },
  {
    "cell": null,
    "code": "B904",
    "end_location": {
      "column": 10,
      "row": 2258
    },
    "filename": "C:\\Users\\wayne\\GitHub\\Python\\Projects\\Ancestry\\database.py",
    "fix": null,
    "location": {
      "column": 9,
      "row": 2254
    },
    "message": "Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling",
    "noqa_row": 2254,
    "url": "https://docs.astral.sh/ruff/rules/raise-without-from-inside-except"
  },
  {
    "cell": null,
    "code": "B904",
    "end_location": {
      "column": 10,
      "row": 2301
    },
    "filename": "C:\\Users\\wayne\\GitHub\\Python\\Projects\\Ancestry\\database.py",
    "fix": null,
    "location": {
      "column": 9,
      "row": 2297
    },
    "message": "Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling",
    "noqa_row": 2297,
    "url": "https://docs.astral.sh/ruff/rules/raise-without-from-inside-except"
  },
  {
    "cell": null,
    "code": "B904",
    "end_location": {
      "column": 10,
      "row": 2307
    },
    "filename": "C:\\Users\\wayne\\GitHub\\Python\\Projects\\Ancestry\\database.py",
    "fix": null,
    "location": {
      "column": 9,
      "row": 2303
    },
    "message": "Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling",
    "noqa_row": 2303,
    "url": "https://docs.astral.sh/ruff/rules/raise-without-from-inside-except"
  },
  {
    "cell": null,
    "code": "B904",
    "end_location": {
      "column": 10,
      "row": 2351
    },
    "filename": "C:\\Users\\wayne\\GitHub\\Python\\Projects\\Ancestry\\database.py",
    "fix": null,
    "location": {
      "column": 9,
      "row": 2347
    },
    "message": "Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling",
    "noqa_row": 2347,
    "url": "https://docs.astral.sh/ruff/rules/raise-without-from-inside-except"
  },
  {
    "cell": null,
    "code": "B904",
    "end_location": {
      "column": 96,
      "row": 1882
    },
    "filename": "C:\\Users\\wayne\\GitHub\\Python\\Projects\\Ancestry\\main.py",
    "fix": null,
    "location": {
      "column": 9,
      "row": 1882
    },
    "message": "Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling",
    "noqa_row": 1882,
    "url": "https://docs.astral.sh/ruff/rules/raise-without-from-inside-except"
  }
]